Library/Homebrew/dev-cmd/update-python-resources.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def update_python_resources_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `update-python-resources` [<options>] <formula>
+
+        Update versions for PyPI resource blocks in <formula>.
+      EOS
+      switch "-p", "--print-only",
+             description: "Print the updated resource blocks instead of changing the formula."
+      named :formula
+    end
+  end
+
+  def update_python_resources
+    update_python_resources_args.parse
+
+    formula = args.formulae.first
+    pypi_name = if formula.stable.url.start_with?("https://files.pythonhosted.org/packages/")
+      File.basename(formula.stable.url).match(/^(.+)-[a-z0-9.]+$/)[1]
+    else
+      formula.name
+    end
+
+    pypi_resources = formula.resources.select do |resource|
+      resource.url.start_with?("https://files.pythonhosted.org/packages/") && resource.name != "homebrew-virtualenv"
+    end
+
+    odie "No PyPI resources found in \"#{formula.name}\"" if pypi_resources.empty?
+
+    odie "\"python@3.8\" must be installed" unless Formula["python@3.8"].any_version_installed?
+
+    odie "\"pipgrip\" must be installed (`pip3 install pipgrip`)" unless File.exist?("#{HOMEBREW_PREFIX}/bin/pipgrip")
```

Comment:
We can use `brew install` here now that https://github.com/Homebrew/homebrew-core/pull/58380 is merged.